test/services/parcelActionsService.test.js

```diff
@@ -1,17 +1,94 @@
+const mockParcelsToReturn = [{
+  ref: 'SS12345678',
+  SSSI: false
+},
+{
+  ref: 'XX12345678',
+  SSSI: true
+}]
+
+jest.mock('../../server/services/parcelService', () => {
+  return {
+    getByRef: jest.fn((parcelRef) => {
+      return mockParcelsToReturn.find(x => x.ref === parcelRef)
+    })
+  }
+})
+
+jest.mock('../../server/services/actionsService', () => {
+  return {
+    get: jest.fn(() => {
```

Comment:
the action service get function is marked as async, but the mock is returning an array

Reply:
Fixed now, plus a further bug picked up once the tests were running correctly.